akka-remote/src/main/protobuf/WireFormats.proto

```diff
@@ -135,3 +145,81 @@ message AddressData {
   required uint32 port = 3;
   optional string protocol = 4;
 }
+
+/**
+ * java.util.concurrent.TimeUnit enum
+ */
+enum TimeUnit {
+  NANOSECONDS = 1;
+  MICROSECONDS = 2;
+  MILLISECONDS = 3;
+  SECONDS = 4;
+  MINUTES = 5;
+  HOURS = 6;
+  DAYS = 7;
+}
+
+message FiniteDuration {
+  required int64 value = 1;
+  required TimeUnit unit = 2;
+}
```

Comment:
Looking a bit closer at it, that is `Duration` not `FiniteDuration` so were safe here.